content/en/docs/setup/upgrade/canary/index.md

```diff
@@ -123,34 +116,70 @@ The output confirms that the pod is using `istiod-canary` revision of the contro
 
 ## Stable revision labels (experimental)
 
-Manually relabeling namespaces when moving them to a new revision can be tedious and error-prone. [Revision tags](/docs/reference/commands/istioctl/#istioctl-experimental-tag) are a solution to this. [Revision tags](/docs/reference/commands/istioctl/#istioctl-experimental-tag) are stable identifiers that point to revisions and can be used to avoid relabeling namespaces. Rather than relabeling the namespace, a mesh operator can simply change the tag to point to a new revision. All namespaces with that tag will be updated at the same time.
+Manually relabeling namespaces when moving them to a new revision can be tedious and error-prone.
+[Revision tags](/docs/reference/commands/istioctl/#istioctl-experimental-tag) solve this problem.
+[Revision tags](/docs/reference/commands/istioctl/#istioctl-experimental-tag) are stable identifiers that point to revisions and can be used to avoid relabeling namespaces. Rather than relabeling the namespace, a mesh operator can simply change the tag to point to a new revision. All namespaces labeled with that tag will be updated at the same time.
```

Comment:
```suggestion
[Revision tags](/docs/reference/commands/istioctl/#istioctl-experimental-tag) solve this problem. Revision tags are stable identifiers that point to revisions and can be used to avoid relabeling namespaces. Rather than relabeling the namespace, a mesh operator can simply change the tag to point to a new revision. All namespaces labeled with that tag will be updated at the same time.
```

They're graduating from Experimental in 1.11 yes?  Link may need updating too

Reply:
Link is definitely wrong, it's currently `https://istio.io/latest/docs/reference/commands/istioctl/#istioctl-experimental-revision-tag`

Reply:
Good catch! Fixed the link.